client/src/components/Head.css

```diff
@@ -15,3 +15,10 @@
     font-size: 2.5vw;
 }
 
+.editAccountLink:hover {
+    text-decoration: none;
+}
+
+.dropDownItem:active {
+    background-color: white;
+}
```

Comment:
how did you think of this? :)

Reply:
I always try that stunt with reactstrap, never managed it, and I couldn't even find stuff on google about it

Reply:
Mr. Google :)